Kickstarter-iOS/Views/PledgeAddNewCardView.swift

```diff
@@ -0,0 +1,123 @@
+import Foundation
+import Library
+import Prelude
+import UIKit
+
+protocol PledgeAddNewCardViewDelegate: AnyObject {
+  func pledgeAddNewCardViewDidTapAddNewCard(_ view: PledgeAddNewCardView)
+}
+
+final class PledgeAddNewCardView: UIView {
+  private lazy var addNewCardImageView: UIImageView = {
+    UIImageView(image: UIImage.init(named: "icon--add")?.withRenderingMode(.alwaysOriginal))
```

Comment:
Is there a reason why we are overriding the renderingMode here instead of in the `image set`?

Reply:
Good question... I guess since it's highly unlikely we'll ever reuse this view with a different tint it makes sense to set it as an original image in the asset folder 👍